application/app/components/fiches/ficheEtablissement/ProtectionCard.tsx

```diff
@@ -1,12 +1,36 @@
+import { useState } from 'react';
+
+import { Collapsible, CollapsibleContent, CollapsibleTrigger } from '@/components/ui/collapsible';
 import { CircleAlert } from 'lucide-react';
 
-const PROTECTION_TEXT = 'Une partie de cet établissement est située en zone protégée';
+const PROTECTION_TEXT = 'Une partie de cet établissement est située en zone protégée.';
+const INFO_TRIGGER_TEXT = "Plus d'info";
+const INFO_HIDE_TEXT = "Moins d'info";
+const INFO_TEXT =
+	"Cette zone classée bâtiment de France est une zone qui se trouve à proximité d'un monument ou d'un site historique protégé par la loi. Elle délimite des secteurs aux abord de sites historiques protégés, établissant un périmètre de 500m autour de chaque monument concerné. ";
 
 const ProtectionCard: React.FC = () => {
+	const [triggerText, setTriggerText] = useState<string>(INFO_TRIGGER_TEXT);
+
 	return (
 		<div className={'mb-4 flex gap-4 rounded-md bg-orange p-2'}>
-			<CircleAlert color='orange' size={40} />
-			<p className='text-sm font-normal text-blue'>{PROTECTION_TEXT}</p>
+			<CircleAlert color='orange' size={40} className='min-h-7 min-w-7 sm:h-10 sm:w-10' />
```

Comment:
```suggestion
			<CircleAlert color='orange' size={26} className='shrink-0' />
```

C'est le container flex qui fait changer la taille de l'icone, je pense qu'on devrait juste ne pas shrink cette icone. Aussi la taille ne correspond pas à celle du figma, tu peux mettre 26 car le cercle du svg ne fait pas  24 si tu mets 24...